src/test/groovy/pl/allegro/tech/discovery/consul/recipes/watch/ConsulWatcherTest.groovy

```diff
@@ -208,25 +209,70 @@ class ConsulWatcherTest extends Specification {
         await().atMost(Duration.FIVE_SECONDS).until({ consumedMessages == ['success'] })
     }
 
+    def "should reset index on failure"() {
+        given:
+        consul.resetAll()
```

Comment:
Why only in this test method `resetAll` is called? Is it required, if yes then why in other tests not?

Reply:
deleted, i added it here my mistake